container-images/mds-metrics-sheet/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8-alpine
```

Comment:
`mds-metrics-sheet` is a little different because it doesn't have a server component. They are just scheduled lambdas.

Reply:
Should run just fine as a cron job in k8s

Reply:
Sure, but there is no `server.ts` in this package presently.

Reply:
ah, gotcha.  is metrics-log.js the correct entry point?

Reply:
I think potentially there are 2 different entry points. Neil is probably most familiar.

Reply:
Updated the entrypoint to use the correct file.